storybook/stories/core/components/Form/GeneralInformation.story.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <oxd-form>
+    <oxd-grid :cols="4">
+      <oxd-grid-item>
+        <oxd-form-row>
+          <oxd-form-group label="Oraganization Name">
+            <oxd-input />
+          </oxd-form-group>
+        </oxd-form-row>
+      </oxd-grid-item>
+      <oxd-grid-item>
+        <oxd-form-row>
+          <oxd-form-group label="Number of Employees">
+            <oxd-input />
+          </oxd-form-group>
+        </oxd-form-row>
+      </oxd-grid-item>
+      <oxd-grid-item>
+        <oxd-form-row>
+          <oxd-form-group label="Registation Number">
+            <oxd-input />
+          </oxd-form-group>
+        </oxd-form-row>
+      </oxd-grid-item>
+      <oxd-grid-item>
+        <oxd-form-row>
+          <oxd-form-group label="Tax Id">
+            <oxd-input />
+          </oxd-form-group>
+        </oxd-form-row>
+      </oxd-grid-item>
+      <oxd-grid-item>
+        <oxd-form-row>
+          <oxd-form-group label="Phone">
+            <oxd-input />
+          </oxd-form-group>
+        </oxd-form-row>
+      </oxd-grid-item>
+      <oxd-grid-item>
+        <oxd-form-row>
+          <oxd-form-group label="Fax">
+            <oxd-input />
+          </oxd-form-group>
+        </oxd-form-row>
+      </oxd-grid-item>
+      <oxd-grid-item>
+        <oxd-form-row>
+          <oxd-form-group label="Email">
+            <oxd-input />
+          </oxd-form-group>
+        </oxd-form-row>
+      </oxd-grid-item>
+      <oxd-grid-item> </oxd-grid-item>
+      <oxd-grid-item>
+        <oxd-form-row>
+          <oxd-form-group label="Address Street 1">
+            <oxd-input />
+          </oxd-form-group>
+        </oxd-form-row>
+      </oxd-grid-item>
+      <oxd-grid-item> </oxd-grid-item>
+      <oxd-grid-item>
+        <oxd-form-row>
+          <oxd-form-group label="Address Street 2">
+            <oxd-input />
+          </oxd-form-group>
+        </oxd-form-row>
+      </oxd-grid-item>
+      <oxd-grid-item> </oxd-grid-item>
```

Comment:
Because of each grid item padding and height, defining an empty grid item will not work properly in small viewports